app/helpers/collections_helper.rb

```diff
@@ -103,4 +103,25 @@ def collection_item_display_title(collection_item)
       ts('Deleted or unknown item')
     end
   end
+
+  def collection_item_approval_options_label(options = {})
+    if options[:actor] == "user" && (options[:item_type] == "bookmark" || options[:item_type] == "work")
+      t("collections_helper.collection_item_approval_options_label.user.#{options[:item_type]}")
```

Comment:
I was  having a hell of a time finding any information on how to use Rails i18n in helpers -- all my Google results were telling me about i18n helpers, not _using_ it in helper files -- so I just threw my hands up in defeat and made something up on the spot. Do you know of something I could read?

Reply:
I haven't read much about best practices. I mostly just use a modified version of [this monkeypatch](http://www.unixgods.org/Rails/where_is_Rails_trying_to_lookup_L10N_strings.html) to experiment. (The actual information on that page might be somewhat outdated, though, because the information about relative scopes in controllers looks wrong.)

Helpers don't affect the i18n scope used for lazy key lookup, so using `t(".x")` in a helper file is exactly the same as using it in the view file that calls the helper. And view files pick a scope based on the current template name. (Which in this case is `[:collection_items, :collection_item_controls]` because these functions are exclusively used in `/app/views/collection_items/_collection_item_controls.html.erb`.)